src/cpu/vc.rs

```diff
@@ -594,14 +594,24 @@ pub fn vc_cpuid(leaf: u32, subleaf: u32) -> (u32, u32, u32, u32) {
             } else {
                 ecx &= !BIT!(27);
             }
+            // Check for RDRAND bit and set ecx
```

Comment:
RDRAND (and RDSEED) are guaranteed available on any platform that supports SNP. So you should unconditionally set the RDRAND (and RDSEED) bits instead of relying on the hypervisor to supply them.